### DISCUSSION TITLE
Latent Live Preview with the API

### DISCUSSION THREAD

**Discussion by EmmaShredda** [Comfy-Org/ComfyUI#4460] [Q&A]
Hey everyone! I'm building out an application that interfaces with Comfy's API, I have a full communication system in place, can receive progress percentage, and final images. But Is there a way to send the latent previews / live previews over through the API? I've really tried to do my own searching but I've hit a wall with this one. Thanks :)

**Comment by ltdrdata** [comment]
You have to utilize websocket.
https://github.com/comfyanonymous/ComfyUI/issues/952

**Reply by EmmaShredda** [reply]
Okay great so it is possible then. Is there a certain condition I have to call or a specific route? I've followed the example scripts but am only receiving final images. And back in comfy I do have previews on, although I'm setting that through the manager.

**Reply by ltdrdata** [reply]
> Okay great so it is possible then. Is there a certain condition I have to call or a specific route? I've followed the example scripts but am only receiving final images. And back in comfy I do have previews on, although I'm setting that through the manager.

The live preview is continuously updating, so you need to poll periodically while the prompt is running.

**Reply by n0valis** [reply]
> The live preview is continuously updating, so you need to poll periodically while the prompt is running.

How does this work? Is there some code example for this somewhere on the internet?

**Comment by PGCRT** [comment]
@EmmaShredda did you find a solution or documentation for this? 

I'm having the same issue, I can fetch the generated image but not the intermediate latent preview frames
